templates/maas/index.html

```diff
@@ -1,1060 +1,700 @@
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/ab9074df-logo-tele2.png"
-               alt="tele2 logo" />
-        </div>
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/56c03384-logo-nec.png"
-               alt="nec logo" />
-        </div>
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/822701bb-logo-tmobile.png"
-               alt="tmobile logo" />
-        </div>
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/673fa219-logo-ntt.png"
-               alt="ntt logo" />
-        </div>
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/022f6f41-logo-bt.png" alt="bt logo" />
-        </div>
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/a59ea622-logo-liberty-global.png"
-               alt="liberty-global logo" />
-        </div>
-        <div class="col-small-1 col-medium-1 col-2">
-          <img src="https://assets.ubuntu.com/v1/9bb88cf6-logo-barclays+copy.png"
-               alt="barclays logo" />
-        </div>
-      </div>
-    </noscript>
-  </section>
-
-  <script>
-    function fragmentFromString(htmlString) {
-      var temp = document.createElement('template');
-      temp.innerHTML = htmlString;
-      return temp.content;
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_1' -%}
+      <h3 class="p-heading--5">Automated hardware lifecycle management</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_1' -%}
+      <p>
+        Automatically discovers, inventories, commissions, and deploys physical servers. Through the server's baseboard management controller (BMC), MAAS can manage remote power and other out-of-band controls.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_2' -%}
+      <h3 class="p-heading--5">Infrastructure-as-code automation</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_2' -%}
+      <p>
+        Use a cloud-like API to programmatically manage the entire hardware lifecycle as code. MAAS integrates with DevOps tools like Terraform, Juju and Ansible to automate, orchestrate, and model entire physical infrastructure deployments from a single workflow.
+      </p>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_title_3' -%}
+      <h3 class="p-heading--5">Integrated hardware diagnostics</h3>
+    {%- endif -%}
+
+    {%- if slot == 'list_item_description_3' -%}
+      <p>
+        Execute extensive pre-commissioning tests to establish a performance baseline and verify the integrity of critical components, ensuring only compliant and healthy machines are provisioned.
+      </p>
+      <p>
+        <a href="https://ubuntu.com/engage/maas-hardware-testing">Learn how HW can be tested at scale in data centers ›</a>
```

Comment:
Elsewhere (in this file and most others) we seem to use `&nbsp;&rsaquo;` instead of ` ›`. Maybe it's fine but mentioning it in case you want to change these (there are a few cases of this in the file, but I've only left this comment).